### ISSUE TITLE
Add structures support

### ISSUE THREAD

**Issue by YourCoalAlt** [YourCoalAlt/SeedSearcherStandaloneTool#2]
Find things like villages, ocean monuments, etc

**Comment by Zodsmar** [comment]
This killed me for weeks and I never got it working. Hoping you get structure searching working :)

**Comment by YourCoalAlt** [comment]
It 's having a rough time working, I've gotten stuck on it now. It appears to only work within the spawn chunks, and then the behavior differs each seed. Sometimes it finds a structure up to 512 blocks out of the range of the search and it will make it valid, or the structure doesn't exist at all but still says the seed is valid. I haven't figured out another way yet, though I've only dug through the Amidst code. At this point, I may fork Amidst myself and try to implement features into it that will make it work for the Seed Search tool, but I have also thought about just looking into using the Minecraft code directly.

(I don't think I have submitted my latest changes yet to Github since I didn't intend on making a new "release" yet)

**Comment by Zodsmar** [comment]
Yeah, so what you're explaining is similar to the situation I had where it was working but very randomly and almost sporadic. I have thought about looking into the minecraft code directly too. I got quite busy with other projects so I don't have much time to dedicate to this but I do still definitely want to get a few ideas added but without structures, I'm kinda at a standstill. I am going to keep an eye out on your work and I can try to assist if I notice something might work.

**Comment by YourCoalAlt** [comment]
I began to add some GUI interface to it, also with some new searching methods, if you wish to check them out I just went ahead and pushed my latest code from two weeks ago to Github in case anyone wanted to know what I was doing. (You need to check Dev Mode and check 'Find Structures' to get the 'structures' tab to be available)
https://github.com/YourCoalAlt/SeedSearcherStandaloneTool/releases/tag/0.3.0-alpha.1
